src/bencode.rs

```diff
@@ -1,7 +1,7 @@
 use std::collections::BTreeMap;
 
 #[derive(Clone, Hash, Eq, PartialEq, Ord, PartialOrd)]
-pub struct BencodableByteString(pub Vec<u8>);
+pub struct BencodableByteString(Vec<u8>);
```

Comment:
I wanted to offer references instead of direct access, but I am not sure that matters.